spec/lib/metasploit/framework/login_scanner/mssql_spec.rb

```diff
@@ -35,7 +35,6 @@
   it_behaves_like 'Metasploit::Framework::LoginScanner::Base',  has_realm_key: true, has_default_realm: true
   it_behaves_like 'Metasploit::Framework::LoginScanner::RexSocket'
   it_behaves_like 'Metasploit::Framework::LoginScanner::NTLM'
-  it_behaves_like 'Metasploit::Framework::Tcp::Client'
```

Comment:
Would love feedback on this - removed the shared example because it didn't seem appropriate to used that shared example since that's no longer how this works. If this _should_ be compatible with that, or this is now missing test coverage that should otherwise be included, please let me know and I'd be happy to tweak/discuss

Reply:
Seems legit to nuke